astropy/units/astrophys.py

```diff
@@ -123,9 +123,7 @@
 # This is not formally a unit, but is used in that way in many contexts, and
 # an appropriate equivalency is only possible if it's treated as a unit (see
 # https://arxiv.org/pdf/1308.4150.pdf for more)
-# Also note that h or h100 or h_100 would be a better name, but they either
-# conflict or have numbers in them, which is apparently disallowed
-def_unit(['littleh'], namespace=_ns, prefixes=False,
+def_unit(['littleh', 'h100'], namespace=_ns, prefixes=False,
```

Comment:
Huh, so numbers are allowed now?

Reply:
I guess so. :).

Reply:
@pllim is right that numbers should not be there, as they are normally interpreted as a power: 
```
In [2]: u.Unit('cm2')
Out[2]: Unit("cm2")

In [3]: u.Unit('cm2') == u.cm**2
Out[3]: True
```

Do try whether combinations like `u.Unit('h100 Mpc')` work.

Reply:
Egads. That produces $hour^{100} * Mpc$

Reply:
Well. Closing.